docs/src/main/paradox/docs/getting-started/running-nexus/configuration/index.md

```diff
@@ -0,0 +1,91 @@
+# Nexus configuration
+
+Nexus Delta service can be highly customized using @link:[configuration file(s)](https://github.com/BlueBrain/nexus/blob/master/delta/app/src/main/resources/app.conf){ open=new }. Many things can be adapted to your deployment needs: port where the service is running, timeouts, the database you decide to support, pagination defaults, etc. 
+
+There are 2 ways to modify the default configuration:
+
+- Using JVM properties as arguments when running the service: -D`{property}`. For example: `-Dapp.http.interface="127.0.0.1"`.
+- Using @link:[FORCE_CONFIG_{property}](https://github.com/lightbend/config#user-content-optional-system-or-env-variable-overrides){ open=new }
+  environment variables. In order to enable this style of configuration, the JVM property
+  `-Dconfig.override_with_env_vars=true` needs to be set. Once set, a configuration flag can be overridden. For example: `CONFIG_FORCE_app_http_interface="127.0.0.1"`.
+  
+In order to successfully run Nexus Delta there is a minimum set of configuration flags that need to be specified
+
```

Comment:
A word about the recommended JVM options here ?

Reply:
What do you mean? You can write them down as a comment here so I can add them

Reply:
JAVA_OPTS: `-Xms4g -Xmx4g`